spacemesh/v1/admin.proto

```diff
@@ -12,4 +12,7 @@ service AdminService {
 
   // Recovers from the provided checkpoint data.
   rpc Recover(RecoverRequest) returns (google.protobuf.Empty);
+
+  // Events that are relevant for node operator
+  rpc EventsStream(EventStreamRequest) returns (stream Event);
```

Comment:
I thought there will be a separate stream of events for each in-node process, to not mix events coming from, say atxBuilder building process and beacon.

Will smapp show the record of all previous events? I imagine that, if the atxBuilder emitted `EventPoetWaitChallenge` then Smapp should show a big label "waiting for poet proof" for ~2 weeks regardless of any other event showing about after it.

Reply:
> I thought there will be a separate stream of events for each in-node process, to not mix events coming from, say atxBuilder building process and beacon.

i can't just justify doing that. there is a limited number of events that are interesting for node admin. if we see that they grow to some huge number then certainly

> Will smapp show the record of all previous events? I imagine that, if the atxBuilder emitted EventPoetWaitChallenge then Smapp should show a big label "waiting for poet proof" for ~2 weeks regardless of any other event showing about after it.

i imagine that there will be a log events. like
- waiting for poet
- published proposal
- published proposal

so maybe waiting for poet won't be the last one. @brusherru please consider

Reply:
what about the scenario where Smapp is opened and connects to an already-running go-sm node? it could be a long time in between events in the stream. don't we need a separate, non-stream endpoint to query the current/latest status?

Reply:
i realized that smesher will miss some events today, i wont be adding status but i will cache latest events